src/web/app/src/components/Posts/index.tsx

```diff
@@ -21,12 +21,12 @@ const useStyles = makeStyles((theme: Theme) =>
       width: '100%',
     },
     error: {
-      color: '#B5B5B5',
+      color: '#4d4d4d',
```

Comment:
Can we try to add this our themes so we don't have to hardcode colours? Same for `errorIcon`.

Reply:
I am using colour from themes.